### PR TITLE
Progress modules to dashboard 60

### LINKED ISSUE #60: Reference ProgressModule component in Dashboard
Need to Add four ProgressModule components to the Dashboard component. They also need to receive props of some kind.

### PR DESCRIPTION
**Pull request by lanelps**
closes #60

### CODE REVIEW

File: client/components/Dashboard.jsx [hunk 1]
```diff
@@ -1,10 +1,26 @@
 import React from 'react'
 
+import ProgressModule from './ProgressModule'
 
 export default function Dashboard() {
+  const placeHolderProps = {
+    tally: 20
```
Comment (emilyparkes) [inline client/components/Dashboard.jsx]: could this be changed to percentage? Tally doesn't quite make sense in this context.

File: client/components/ProgressModule.jsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import CircularProgressbar from 'react-circular-progressbar'
 
 export default function ProgressModule(props) {
-  const percentage = props.placeHolderTallyNumber
+  const percentage = props.placeHolderProps.tally
```
Comment (emilyparkes) [inline client/components/ProgressModule.jsx]: Again here. Why is it tally and then named to percentage?

**Review by emilyparkes — CHANGES_REQUESTED** [review]

**Review by emilyparkes — APPROVED** [review]

**Comment by chrislaupama** [comment]
@lanelps this isn't working on my end.

https://github.com/harakeke-2019/EDAMAME/commit/cf2f5edcbc08d3371999048ad7fe70e10d0eeaf2

`import CircularProgressbar from 'react-circular-progressbar'`

**Comment by chrislaupama** [comment]
@lanelps I'm having an issue with https://github.com/harakeke-2019/EDAMAME/commit/cf2f5edcbc08d3371999048ad7fe70e10d0eeaf2 running.

It looks like `import CircularProgressbar from 'react-circular-progressbar'` isn't importing anything.

**Pull request merged**